### DISCUSSION TITLE
Functional recommendations in AndroidStudio

### DISCUSSION THREAD

**Discussion by Chen-Xi-g** [community/community#14909] [Copilot Conversations] [answered]
The hard-coded string is converted to @ string resources and translated into resource names according to the content

**Comment by Kek5chen** [comment, marked as answer]
GitHub Copilot only helps you to write the code and that only in the file you are currently editing. This is not what copilot was designed for. Think of the name copilot, it writes the strings and you put them into the resources. You're the pilot after all.

**Comment by Chen-Xi-g** [comment]
@Kek5chen Thank you. You're absolutely right.

**Reply by Kek5chen** [reply]
If you think that my answer answered your question it would be nice if you could mark it as the answer. Thanks
